site/en/docs/web-platform/region-capture/index.md

```diff
@@ -7,7 +7,7 @@ authors:
   - beaufortfrancois
   - eladalon
 date: 2022-06-21
-updated: 2022-08-04
+updated: 2022-10-04
```

Comment:
Let's not merge the PR until the rollout reaches 100%.

Reply:
Yes. See https://github.com/GoogleChrome/developer.chrome.com/pull/3898#issue-1396197102

Reply:
I'd missed that. Thanks for pointing it out.